Formula/bucklespring.rb

```diff
@@ -0,0 +1,23 @@
+class Bucklespring < Formula
+  desc "Nostalgia bucklespring keyboard sound"
+  homepage "https://github.com/zevv/bucklespring"
+  url "https://github.com/zevv/bucklespring/archive/v1.4.0.tar.gz"
+  sha256 "b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348"
+  license "GPL-2.0-only"
+
+  depends_on "alure" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    inreplace "Makefile", "-Wall -Werror", "-Wall"
+    ENV["PATH_AUDIO"] = prefix/"wav"
+    system "make"
+    (lib/"pkgconfig").install Dir["mac/lib/pkgconfig/*"]
```

Comment:
```suggestion
    cp Formula["pkg-config"].opt_lib/"alure.pc" "mac/lib/pkgconfig/"
```
Sorry, I misunderstood what you were trying to do here. @SMillerDev's suggestion for this is the correct one.